src/assets/javascript/script.js

```diff
@@ -20,4 +20,52 @@ const navToggle = () => {
     });
 }
 
-navToggle();
\ No newline at end of file
+const getRepoMembers = async () =>{
+    const members = await (await fetch("https://api.github.com/orgs/chryz-hub/members", {
+        headers: new Headers({
+            "Authorization": 'token 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'
+        })
+    })).json();
+    return members;
+}
+
+const getMembersInfo = async (memberLogin) =>{
+    const memberInfo = await (await fetch(`https://api.github.com/users/${memberLogin}`, {
+        headers: new Headers({
+            "Authorization": 'token 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'
```

Comment:
I think it is not recommended to add the authorization token in plain text into the http call header 🤔

Reply:
Of course. Like I mentioned, it's a dummy example token, just to show that the token is needed. We should store it as a secret which the community mods should have access to.
Public APIs are accessible without auth token, but while testing this I observed there was a rate limit for unauthorised calls. For authorised calls, there is no rate limit.